getting-started/local-node/setting-up-a-node.md

```diff
@@ -68,48 +108,20 @@ Then, you will want to run the node in dev mode using the following command:
 !!! note
     For people not familiar with Substrate, the `--dev` flag is a way to run a Substrate-based node in a single node developer configuration for testing purposes. You can learn more about `--dev` in [this Substrate tutorial](https://substrate.dev/docs/en/tutorials/create-your-first-substrate-chain/interact).
 
-You should see an output that looks like the following, showing that blocks are being produced:
+You should see an output that looks like the following, showing an idle state waiting for blocks to be produced:
 
 ![Output shows blocks being produced](/images/setting-up-a-node/setting-up-node-3b.png)
 
-The local standalone Moonbeam node provides two RPC endpoints:
-
- - HTTP: `http://127.0.0.1:9933`
- - WS: `ws://127.0.0.1:9944`
-
-## Getting Started with Docker
-
-An alternative to the steps higlighted before is to use Docker to run a pre-build binary. Doing so, you prevent having to install Substrate and all the dependencies, and you can skip the building the node process as well. The only requirement is to have Docker installed, and then you can execute the following command to download the corresponding image:
+For more information on some of the flags and options used in the example, check out [Common Flags and Options](#common-flags-and-options). If you want to see a complete list of all of the flags, options, and subcommands, open the help menu by running:
```

Comment:
We should also mention the `moonbeam --help` flag. There is a lot to learn there.